plugin/evm/vm.go

```diff
@@ -122,9 +122,9 @@ const (
 	codecVersion         = uint16(0)
 	secpFactoryCacheSize = 1024
 
-	decidedCacheSize    = 100
-	missingCacheSize    = 50
-	unverifiedCacheSize = 50
+	decidedCacheSize    = 10 * units.MiB
+	missingCacheSize    = 5 * units.MiB
```

Comment:
the missing cache is not byte based - as it does not contain any blocks

Reply:
Will update